configure

```diff
@@ -30,6 +30,14 @@ isopenbsd() {
 	uname -s | grep OpenBSD >/dev/null
 }
 
+issolaris() {
+	[ "`uname -s`" = SunOS ]
+}
+
+isgunld() {
```

Comment:
i think this should be named GNU, not GUN

but then, wouldn't it be better to check if the linker supports the feature you want ?

also, since we never call ld directly, it doesnt actually matter what ld is. we need to know the properties of the linker called by compiler...

Reply:
btw, someone might use a crosscompiler, and then running "ld" is entirely counter-productive...

i also do not like the uname check, but i guess i can't say nothing against it since i did it myself :/

Reply:
I will fix the typo in next commit

Reply:
I check for the GNU ld because some GNU ld specific options `--no-as-needed` and `--soname` are used in `Makefile`